configuration_go/abstr/kubernetes/thanos/compactor/compactor.go

```diff
@@ -15,55 +16,54 @@ const (
-	TracingConfig                      string        `opt:"tracing.config"`
-	TracingConfigFile                  string        `opt:"tracing.config-file"`
-	Version                            bool          `opt:"version,noval"`
-	Wait                               bool          `opt:"wait,noval"`
-	WaitInterval                       time.Duration `opt:"wait-interval"`
-	WebDisable                         bool          `opt:"web.disable"`
-	WebDisableCors                     bool          `opt:"web.disable-cors"`
-	WebExternalPrefix                  string        `opt:"web.external-prefix"`
-	WebPrefixHeader                    string        `opt:"web.prefix-header"`
-	WebRoutePrefix                     string        `opt:"web.route-prefix"`
+	BlockFilesConcurrency              int                         `opt:"block-files-concurrency"`
+	BlockMetaFetchConcurrency          int                         `opt:"block-meta-fetch-concurrency"`
+	BlockViewerGlobalSyncBlockInterval time.Duration               `opt:"block-viewer.global.sync-block-interval"`
+	BlockViewerGlobalSyncBlockTimeout  time.Duration               `opt:"block-viewer.global.sync-block-timeout"`
+	BucketWebLabel                     string                      `opt:"bucket-web-label"`
+	CompactBlocksFetchConcurrency      int                         `opt:"compact.blocks-fetch-concurrency"`
+	CompactCleanupInterval             time.Duration               `opt:"compact.cleanup-interval"`
+	CompactConcurrency                 int                         `opt:"compact.concurrency"`
+	CompactProgressInterval            time.Duration               `opt:"compact.progress-interval"`
+	ConsistencyDelay                   time.Duration               `opt:"consistency-delay"`
+	DataDir                            string                      `opt:"data-dir"`
+	DeduplicationFunc                  string                      `opt:"deduplication.func"`
+	DeduplicationReplicaLabel          string                      `opt:"deduplication.replica-label"`
+	DeleteDelay                        time.Duration               `opt:"delete-delay"`
+	DownsampleConcurrency              int                         `opt:"downsample.concurrency"`
+	DownsamplingDisable                bool                        `opt:"downsampling.disable"`
+	HashFunc                           string                      `opt:"hash-func"`
+	HttpAddress                        *net.TCPAddr                `opt:"http-address"`
+	HttpGracePeriod                    time.Duration               `opt:"http-grace-period"`
+	HttpConfig                         string                      `opt:"http.config"`
+	LogFormat                          common.LogFormat            `opt:"log.format"`
+	LogLevel                           common.LogLevel             `opt:"log.level"`
+	MaxTime                            *common.TimeOrDurationValue `opt:"max-time"`
+	MinTime                            *common.TimeOrDurationValue `opt:"min-time"`
```

Comment:
I'm not the biggest fan of naming packages like this (`common`, `utils`, etc). But this is not a blocker at all, we can change it later. I would suggest a `log` and `time` packages.

Reply:
Yup I agree.

Reply:
done